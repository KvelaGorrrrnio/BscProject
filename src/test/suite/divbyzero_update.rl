int a

init: entry
  a /= 4 - (2 * 2)
exit
