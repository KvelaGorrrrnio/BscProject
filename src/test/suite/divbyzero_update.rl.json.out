{ "type" : "error", "position" : { "line" : 4, "column" : 3 }, "message" : "A runtime error occurred at (line 4, column 3):\nDivision by zero." }
